### PR TITLE
Initial commit 

### PR DESCRIPTION
**Pull request by alasarr**
On this PR, I've moved the scripts from: https://github.com/opengeospatial/geoparquet/pull/113

I've also added several improvements:
- Remove poetry, I really like poetry as a developer but I think could be an issue for adoption as not all users use poetry.
- Allow overwriting files.
- Update some dependencies.
- CI for testing.
- CI job to upload several tables to gs://geoparquet. I've added US boundaries.
 
The following files have been created from bigquery and uploaded to `geoparquet` bucket:

https://storage.googleapis.com/geoparquet/geography_usa_state_2019.parquet
https://storage.googleapis.com/geoparquet/geography_usa_county_2019.parquet
https://storage.googleapis.com/geoparquet/geography_usa_censustract_2019.parquet
https://storage.googleapis.com/geoparquet/geography_usa_zcta5_2019.parquet
https://storage.googleapis.com/geoparquet/geography_usa_blockgroup_2019.parquet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,90 @@
-# bigquery-converter
\ No newline at end of file
+# bigquery-converter
+
+Experimental python scripts to import/export [GeoParquet](https://geoparquet.org) files in [Google BigQuery](https://cloud.google.com/bigquery).
+
+## Motivation
+BigQuery supports parquet files, but for tables with geometry data it's not supported. For example, if you try to export a table with a geography column it throws the following error:
```
Comment (cholmes) [inline README.md]:
```suggestion
BigQuery supports Parquet files, but they won't work if you have geometry data. For example, if you try to export a table with a geography column it throws the following error:
```

**Review by cholmes — APPROVED** [review]
Looks good, but I'm not a python dev so only am commenting on the readme - put in a minor language edit.

**Comment by kylebarron** [comment]
> * Remove poetry, I really like poetry as a developer but I think could be an issue for adoption as not all users use poetry

It's fine if you don't use poetry here, but just as a general comment, poetry supports PEP 517, so you don't need poetry installed to install a package that uses poetry. `pip install .` will just work.

**Comment by kylebarron** [comment]
> * Remove poetry, I really like poetry as a developer but I think could be an issue for adoption as not all users use poetry

It's fine if you don't use poetry here, but just as a general comment, poetry supports PEP 517, so you don't need poetry installed to install a package that uses poetry. `pip install .` will just work.

**Comment by alasarr** [comment]
>It's fine if you don't use poetry here, but just as a general comment, poetry supports PEP 517, so you don't need poetry installed to install a package that uses poetry. pip install . will just work.

No idea about it @kylebarron, thanks for letting me know.  I'll add poetry then, if you want to commit that would be great too.

**Pull request merged**